### PR TITLE
using simpler aggregation to get initial list of latest snapshots

### PR DESCRIPTION
**Pull request by mateodelnorte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,55 @@ sourced-repo-mongo
...
+
+sourced-repo-mongo provides a exports Repository, which you can inherit. Doing so for each of your `sourced` entities will provide you a means of getting and persisting entities from mongo: 
+
+```
+const Customer = require('./models/customer');
+const MongoRepository = require('sourced-repo-mongo').Repository;
+const util = require('util');
+
+function CustomerRepository () {
+  MongoRepository.call(this, Trader);
```
Comment (kylestev) [inline README.md]: `s/Trader/Customer/`

**Comment by kylestev** [comment]
Nice. Having more examples like this would make consuming this API more friendly to new users of the `sourced` packages. The lack of documentation and the barebones Market entity in the tests, while suitable for testing, result in a higher barrier to entry.

**Comment by mateodelnorte** [comment]
Yup. Totally understand. I'm working on a site that will house examples of
using a number of these tools together. Happy to keep you up to date, if
you like.

**Comment by kylestev** [comment]
That would be fantastic!

**Pull request closed without merging**